app/community/edit/[id]/edit-community-form.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import React from "react";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useAuth } from "@clerk/nextjs";
+import { useRouter } from "next/navigation";
+import { useSuspenseQuery } from "@tanstack/react-query";
+import { useToast } from "@/hooks/use-toast";
+import { useEditCommunity } from "@/lib/mutations/community-edit";
+import { captureException } from "@/lib/report";
+import { communityInfo } from "@/lib/queries/community";
+import { CommunityForm } from "@/components/features/community/community-form";
+import { communityFormSchema, CommunityFormSchemaType } from "@/types/schemas";
+// TODO: useTranslations
+
+export const EditCommunityForm: React.FC<{ communityId: string }> = ({
+  communityId,
+}) => {
+  const { getToken } = useAuth();
+  const router = useRouter();
+  const { toast } = useToast();
+  const { data: communityData } = useSuspenseQuery(communityInfo(communityId));
```

Comment:
the community admins emails should be fetch from the backend endpoint since email are not on-chain
see how gatekeepers are handled and the zenaoClient.GetCommunityAdministrators func